src/pages/index.js

```diff
@@ -66,20 +111,114 @@ function renderCard(data) {
 /*-------------------------------------------------------------------------------*/
 
 function handleProfileFormSubmit(data) {
-  userInfo.setUserInfo(data);
-  profileFormValidator.resetValidation();
-  profileEditPopup.close();
+  profileEditPopup.renderLoading(true);
+  api
+    .updateUserInfo(data)
+    .then(() => {
+      userInfo.setUserInfo(data);
+    })
+    .then(() => {
+      profileFormValidator.disableSubmit();
+    })
+    .then(() => {
+      profileEditPopup.clearInputs();
+    })
+    .then(() => {
+      profileEditPopup.close();
```

Comment:
If you have several `.then` blocks going one after another, then you can **combine** them together


```js
    .then(() => {
      userInfo.setUserInfo(data);
      profileFormValidator.disableSubmit();
      profileEditPopup.clearInputs();
      profileEditPopup.close();
    })
    .catch((err) => {

```